app/src/main/java/org/jellyfin/androidtv/ui/livetv/LiveTvGuideActivity.java

```diff
@@ -136,26 +139,32 @@ public void onCreate(Bundle savedInstanceState) {
         mSpinner = findViewById(R.id.spinner);
         mSpinner.setVisibility(View.VISIBLE);
 
-        findViewById(R.id.filterButton).setOnClickListener(new View.OnClickListener() {
+        mFilterButton = findViewById(R.id.filterButton);
```

Comment:
You can add the property in the function like so:

```suggestion
        View mFilterButton = findViewById(R.id.filterButton);
```

And then remove the `private View xx` declarations.